packages/forms/stories/8-Radio.stories.tsx

```diff
@@ -30,24 +31,27 @@ export const Default: Story<ICheckboxStoryProps & InputHTMLAttributes<HTMLInputE
     <Grid>
       <Row style={{ minHeight: 450 }}>
         <Col lg={4} offsetLg={4} md={6} offsetMd={3}>
-          <Field>
-            <Radio disabled={disabled} name="example">
-              <Label hidden={isHidden} isRegular={isRegular}>
-                Radio
-              </Label>
-              {showHint && <Hint>Hint</Hint>}
-              {showMessage && <Message validation={validation}>Message</Message>}
-            </Radio>
-          </Field>
-          <Field className="u-mt-xxs">
-            <Radio disabled={disabled} name="example">
-              <Label hidden={isHidden} isRegular={isRegular}>
-                Radio
-              </Label>
-              {showHint && <Hint>Hint</Hint>}
-              {showMessage && <Message validation={validation}>Message</Message>}
-            </Radio>
-          </Field>
+          <MD isBold>Radio group label</MD>
+          <RadioGroup aria-label="radio group label">
```

Comment:
Alternatively, this could be structured as the following:

```jsx
<RadioGroup aria-labelledby="id_1">
  <MD isBold id="id_1">Radio group label</MD>
  {/* radio buttons */}
<RadioGroup>
```

Reply:
Why not `<RadioGroup label="Radio group label">` and then use that for both the display and `aria-label`?

Reply:
No particular reasons not to do that. Before I make the changes I want to understand the suggetion.

Would the API look this this?

```jsx
<RadioGroup label="Choose a plant lifecycle">
  <Field>
    <Radio name="validation-example" value="annual">
      <Label>Annual</Label>
    </Radio>
  </Field>
  <Field>
    <Radio name="validation-example" value="perennial">
      <Label isRegular>Perennial</Label>
    </Radio>
  </Field>
</RadioGroup>
```

And the DOM output look like this?

```html
<div role="group" aria-label="Choose a growth type">
  <div data-garden-id="typography.font">Choose a growth type</div>
  <div>
     <input type="radio">
     <label>Annual</label>
  </div>
  <div>
     <input type="radio">
     <label>Perennial</label>
  </div>
</div>
```

Reply:
Marking as resolved due to change in implementation.